src/library.js

```diff
@@ -501,6 +501,13 @@ $(document).ready(function () {
     var emoji = isSkintoneVariant ? baseEmoji.emoji : currEmoji.emoji;
     var annotation = isSkintoneVariant ? baseEmoji.annotation : currEmoji.annotation;
     var hexcode = currEmoji.hexcode;
+    var zwj = "";
```

Comment:
- find the var name confusing :)
- can you add the links?

... then I think it is good enough to go online. We can style it later.

Reply:
* yes, not sure what the correct terminology should be
* where should we link to?

Reply:
- Maybe `var combination`?
- I would vote to link to us and for ZWJ etc. to a technical resource or https://emojipedia.org/zero-width-joiner/

Reply:
👍👍